schemas/com.shopify/update-product/jsonschema/1-0-0

```diff
@@ -0,0 +1,272 @@
+    },
+    "tags" : {
+      "type" : "string"
+    },
+    "product_type" : {
+      "type" : "string"
+    },
+    "options" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "name" : {
+            "type" : "string"
+          },
+          "id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "product_id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "values" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
+          },
+          "position" : {
+            "type" : "integer",
+            "maximum" : 32767,
+            "minimum" : 0
+          }
+        },
+        "additionalProperties" : false
+      }
+    },
+    "id" : {
+      "type" : "integer",
+      "maximum" : 9223372036854775807,
+      "minimum" : 0
+    },
+    "created_at" : {
+      "type" : "string",
+      "format" : "date-time"
+    },
+    "title" : {
+      "type" : "string"
+    },
+    "template_suffix" : {
+      "type" : "null"
+    },
+    "vendor" : {
+      "type" : "string"
+    },
+    "updated_at" : {
+      "type" : "string",
+      "format" : "date-time"
+    },
+    "images" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "variant_ids" : {
+            "type" : "array",
+            "items" : {
+              "type" : "integer",
+              "maximum" : 9223372036854775807,
+              "minimum" : 0
+            }
+          },
+          "id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "src" : {
+            "type" : "string",
+            "format" : "uri"
+          },
+          "product_id" : {
+            "type" : "integer",
+            "maximum" : 9223372036854775807,
+            "minimum" : 0
+          },
+          "position" : {
+            "type" : "integer",
+            "maximum" : 32767,
+            "minimum" : 0
+          },
+          "created_at" : {
+            "type" : "string",
+            "format" : "date-time"
+          },
+          "updated_at" : {
+            "type" : "string",
+            "format" : "date-time"
+          }
+        },
+        "additionalProperties" : false
+      }
+    },
+    "published_at" : {
+      "type" : [ "string", "null" ],
+      "format" : "date-time"
+    },
+    "variants" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "weight" : {
+            "type" : "number",
+            "minimum" : 0
+          },
+          "inventory_policy" : {
+            "type" : "string",
+            "pattern" : "^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$"
```

Comment:
Unnecessary `pattern`.

Reply:
How to avoid that?

Reply:
No way to avoid it in Schema Guru, unfortunately. We can just remove it manually from schemas.